frontend/src/app/pages/uikit/UiKit.tsx

```diff
@@ -0,0 +1,158 @@
+import { Grid, TextField, Typography } from "@mui/material";
+import Layout from "@components/layout/Layout";
+import { useTranslation } from "react-i18next";
+import FieldHelperText from "@components/fieldHelperText/FieldHelperText";
+import Loading from "@components/loading/Loading";
+import UikitBlock from "@components/uikitBlock/UikitBlock";
+import { ValidationError } from "yup";
+import Link from "@components/link/Link";
+import Spinner from "@components/spinner/Spinner";
+import { useState } from "react";
+import Button from "@components/button/Button";
+import UikitColor from "@components/uikitColor/UikitColor";
+
+export default function UiKit() {
+  const [t] = useTranslation();
+  const [showLoading, setShowLoading] = useState(false);
+
+  const onClickShowLoading = () => {
+    setShowLoading(true);
+
+    setTimeout(() => {
+      setShowLoading(false);
+    }, 3000);
+  };
+
+  // this is for mocking, yup will format the error correctly for you
+  const formErrors: ValidationError[] = [
+    {
+      value: "",
+      path: "username",
+      type: "required",
+      params: {
+        value: "",
+        originalValue: "",
+        label: "login__username",
+        path: "username",
+        spec: {
+          strip: false,
+          strict: false,
+          abortEarly: true,
+          recursive: true,
+          disableStackTrace: false,
+          nullable: false,
+          optional: false,
+          coerce: true,
+          label: "login__username",
+        },
+        disableStackTrace: false,
+      },
+      errors: ["validations__required"],
+      inner: [],
+      name: "ValidationError",
+      message: "validations__required",
+      [Symbol.toStringTag]: "",
+    },
+  ];
+
+  return (
+    <Layout.Container className="mb-lg">
+      <Typography variant="h1">UiKit</Typography>
+      <Typography variant="body1">
+        This is where you can display all your custom components/containers.
```

Comment:
Should this be localized? Same for the text below

Reply:
Vu que c'est uniquement pour l'interne, je voulais pas polluer le fichier de traductions avec des choses du uikit.
T'en penses quoi?